src/components/ControlLink/ControlLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Clickable, ClickableProps } from '../Clickable';
+
+import './ControlLink.scss';
+
+export type ControlLinkProps = ClickableProps & {
+  color?: 'cement' | 'ink';
+  disabled?: boolean;
+  underlined?: boolean;
+};
+
+export const ControlLink: React.FC<ControlLinkProps> = ({
+  className,
+  color = 'cement',
+  disabled,
+  children,
+  underlined,
```

Comment:
I'd make it `underlined` by default, I think we use that version in 90% of the cases.